app/templates/index.hbs

```diff
@@ -1,29 +1,43 @@
 <div class="index"> {{!INDEX}}
 
     {{!HEADER}}
-    <div class="row">{{!HEADER ROW}}
+    <div class="row header-background space-top space-bottom">{{!HEADER ROW}}
         <div class="osf-preprint-header-img"> {{!HEADER}}
             <div class="container"> {{!CONTAINER}}
                 {{!TITLE}}
                 <div class="text-center m-b-lg">
                     <h1 class="osf-preprint-lead">OSF Preprints</h1>
                     <h2>
-                        The <span class="highlight-bold">open</span> preprint server for
-                        <span class="highlight-bold">all</span> academic disciplines
+                        The open preprint server for all academic disciplines
                     </h2>
                 </div>
 
                 {{!SEARCH}}
                 <div id="search" class="space-top space-bottom">
                     {{search-preprints search="search"}}
                     <p>
-                        <span class="highlight">{{model.preprints.length}}</span> preprints as of
+                        {{model.preprints.length}} preprints as of
```

Comment:
Future note, not part of this PR: the server API is paginated, which means that this will usually not be a number > the size of the first page. (we'll want to get the length from the meta section instead)